include/lsst/meas/extensions/shapeHSM/HsmMomentsControl.h

```diff
@@ -183,8 +183,9 @@ class HsmSourceMomentsAlgorithm : public HsmMomentsAlgorithm {
     typedef HsmSourceMomentsControl Control;
 
     /// @brief Initialize with standard field names and customized documentation.
-    HsmSourceMomentsAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema) :
-        HsmMomentsAlgorithm(name, schema, "Source adaptive moments algorithm from HSM"), _ctrl(ctrl)
+    HsmSourceMomentsAlgorithm(Control const & ctrl, std::string const & name, afw::table::Schema & schema,
+                              char const* doc = "Source adaptive moments algorithm from HSM") :  ///< Docstring is exposed for HsmSourceMomentsRoundAlgorithm
```

Comment:
"from HSM" or "from shapeHSM"? Are we referring to the `HSM` paper here, or to this `shapeHSM` package?

Seeing how these come out in the schema, I wonder if "Source adaptive moments via the HSM shape algorithm" (still not sure whether to use `shapeHSM` or `HSM shape` or what) might be better? A user reading the catalog schema is asking "what is this and where did it come from", and "X algorithm from HSM" is answering a different question.

Reply:
Forgive me for intruding, but I thought HSM came from "Hirata, Seljak Moments": https://ui.adsabs.harvard.edu/abs/2003MNRAS.343..459H/abstract   (adaptive moments). I've heard "Hirata, Seljak, Mandelbaum", but I'm not sure that's the case. I'll ask Rachel Mandelbaum.  (This is in response to the comment above "I do wish we had some discussion about what exactly HSM is. ")

Reply:
I don't think `shapeHSM` is going to be particularly useful either, since that's literally the string in the column name. HSM, in practice, refers to a very specific implementation of an algorithm that's not fundamentally different from SDSS shapes. I don't have a better alternative to what you suggested John, so I will go with it. I'll add the reference in the README and create a ticket to put in some module level documentation explaining it in a bit more detail.